lms/views/lti/config.py

```diff
@@ -14,3 +14,63 @@ def config_xml(request):
         "launch_url": request.route_url("lti_launches"),
         "content_item_url": request.route_url("content_item_selection"),
     }
+
+
+@view_config(
+    route_name="lti.v13.canvas.config",
+    renderer="json",
+    request_method="GET",
+)
+def config_json(request):
+    """
+    Return the LTI1.3 configuration for Canvas installs as a json document.
+
+    https://canvas.instructure.com/doc/api/file.lti_dev_key_config.html#anatomy-of-a-json-configuration
+    """
+    return {
+        "title": "Hypothesis",
+        "description": "Hypothesis",
+        "oidc_initiation_url": request.route_url("lti.oidc"),
```

Comment:
Mostly a 1 to 1  translation of the XML document. 

More details here: https://canvas.instructure.com/doc/api/file.lti_dev_key_config.html#anatomy-of-a-json-configuration